renovate.json

```diff
@@ -84,7 +84,7 @@
     {
       "fileMatch": ["\\.*\\.ya?ml$"],
       "matchStrings": [
-        "# renovate: datasource=docker( versioning=(?<versioning>.*?))?\n\\s*-\\s+(?<depName>[^:]+):(?<currentValue>.*)"
+        "# renovate: datasource=docker( versioning=(?<versioning>.*?))?\n\\s+.+:(?<depName>[^:]+):(?<currentValue>.*)"
```

Comment:
I tested this on my fork of this repo and it wasn't picking up the image (bumped it back to 0.26.0 to validate). Not sure if something was wrong on my fork or if something else is happening here...the match works on a regex tester...

Reply:
hmm, i likewise tested with regex101 and saw everything match as expected.

Reply:
added the same regex to renovate in a personal repo with a test yaml and got this in the logs:

```
{
           "depName": "registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller",
            "currentValue": "v0.27.0",
            "datasource": "docker",
            "versioning": "semver",
            "replaceString": "# renovate: datasource=docker versioning=semver\n    default: registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller:v0.27.0",
            "updates": [],
            "packageName": "registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller",
            "warnings": [
              {
                "topic": "registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller",
                "message": "Failed to look up docker package registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller"
              }
            ]
          }
        ],
        "matchStrings": [
          "# renovate: datasource=docker( versioning=(?<versioning>.*?))?\n\\s+.+:(?<depName>[^:]+):(?<currentValue>.*)"
        ],
        "datasourceTemplate": "docker",
        "packageFile": "test.yaml"
}
```

Reply:
Confused what is happening with mine, it did show up in one run in the logs (but errored making the branch due to a depName mismatch). Willing to trust this is g2g and its my fork that is borked.

Separately - might be good to find a way to include this update in the `pepr` dependency group? That way it pops up in the same PR as updating the `package.json` and we can use our normal "waiting on ironbank" label.

Reply:
might not be good if other deps are added to the create.yaml, but I added that file to the file matches for the pepr dep group, so it should combine those updates in same PR.